<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MicroBotix</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/robot</normaloff>:/robot</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="codeInput">
      <property name="tabStopWidth">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="translateButton">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="codeOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="tabStopWidth">
       <number>20</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="mainMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="createNewProgramAction"/>
    <addaction name="openProgramAction"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <addaction name="mainMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createNewProgramAction"/>
   <addaction name="openProgramAction"/>
   <addaction name="saveAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openProgramAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/open</normaloff>:/open</iconset>
   </property>
   <property name="text">
    <string>Ouvrir un programme</string>
   </property>
   <property name="toolTip">
    <string>Ouvrir un programme</string>
   </property>
  </action>
  <action name="createNewProgramAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/create</normaloff>:/create</iconset>
   </property>
   <property name="text">
    <string>Créer un nouveau programme</string>
   </property>
   <property name="toolTip">
    <string>Créer un nouveau programme</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
